frontend/src/hooks/use-handle-click-outside/use-handle-click-outside.hook.tsx

```diff
@@ -1,9 +1,11 @@
 import { RefObject, useEffect } from 'react';
 
-const useHandleClickOutside = (
-  ref: RefObject<HTMLElement>,
-  onClick: () => void,
-): void => {
+type Parameters = {
```

Comment:
Let's name it as in useAppForm hook
```suggestion
type UseHandleClickOutsideArgs = {
```